src/nile/cli.py

```diff
@@ -125,53 +144,39 @@ def setup(signer, network, status_type):
 @click.argument("method", nargs=1)
 @click.argument("params", nargs=-1)
 @click.option("--max_fee", nargs=1)
-@click.option("--track", "-t", "status_type", flag_value="track")
-@click.option("--debug", "-d", "status_type", flag_value="debug")
 @network_option
-def send(signer, address_or_alias, method, params, network, status_type, max_fee=None):
-    """Invoke a contract's method through an Account. Same usage as nile invoke."""
+@watch_option
+@click.option("--simulate", "query", flag_value="simulate")
+@click.option("--estimate_fee", "query", flag_value="estimate_fee")
+def send(
+    signer,
+    address_or_alias,
+    method,
+    params,
+    network,
+    watch_mode,
+    max_fee=None,
+    query=None,
```

Comment:
are these default params needed in click? if they are, i'd add them to `watch_mode`, otherwise we can remove them no?

Reply:
Good catch, they are not needed